scripts/sign_encrypt.py

```diff
@@ -182,15 +182,22 @@ def main():
 
     args = get_args(logger)
 
-    with open(args.key, 'rb') as f:
-        data = f.read()
+    if args.key.startswith('arn'):
```

Comment:
Could you make this `arn:` instead? Rationale: the colon makes it more unlikely to be a PEM file.

Reply:
agreed